parser/src/main/java/com/plhal/ares/dblayer/ParserRepository.java

```diff
@@ -1,10 +1,11 @@
-package com.plhal.ares.model;
+package com.plhal.ares.dblayer;
```

Comment:
Ja nevim, zda si nerozumime nebo co se deje, ze to menite tam a zpatky. Zkusim to tedy napsat naprimo. Nedefinujte stejny balik do dvou modulu (parser, dbLayer). Docela logicky pro parser bude balik pro vsechny tridy `com.plhal.ares.parser` a pro dbLayer tridu zas balik `com.plhal.ares.dblayer`.

Reply:
Já jsem to pochopil, jak to má být. Ale tím, že se ty dva balíky jmenovali stejně, tak když jsem změnil jméno toho jednoho, tak jsem také potvrdil aby se změnil i ten druhý balík a takhle se to měnilo pořád dokola. Každopádně už je to opravené.